packages/sdk/src/channelsWithEntitlements.test.ts

```diff
@@ -675,6 +676,77 @@ describe('channelsWithEntitlements', () => {
         await alice.stopSync()
     })
 
+    test('user with only one entitlement from 3-nested NFT rule data can join channel', async () => {
```

Comment:
The e2e test cases reflect the issue we saw in gamma, with 3 OR'ed rule entitlements.